Source/EasySynth/Private/EXROutput/MoviePipelineEXROutputLocal.cpp

```diff
@@ -129,10 +130,18 @@ bool FEXRImageWriteTaskLocal::WriteToDisk()
 		// Data Window specifies how much data is in the actual file, ie: 1920x1080
 		IMATH_NAMESPACE::Box2i DataWindow = IMATH_NAMESPACE::Box2i(IMATH_NAMESPACE::V2i(0,0), IMATH_NAMESPACE::V2i(Width - 1, Height - 1));
 
+		float BorderRatio = FMath::Clamp((OverscanPercentage / (1.0f + OverscanPercentage)) / 2.0f, 0.0f, 0.5f);
+		int32 OverscanBorderWidth = FMath::RoundToInt(Width * BorderRatio);
+		int32 OverscanBorderHeight = FMath::RoundToInt(Height * BorderRatio);
```

Comment:
Looks const

Reply:
I didn't style-check this file because it is the one taken from the engine. I would rather not touch it, or completely refactor it to match our conventions.